Formula/aurora-cli.rb

```diff
@@ -11,17 +11,16 @@ class AuroraCli < Formula
     sha256 "a97a66254586aace43a789ae73df85781bff46c3d2837a0b9ad8d07bf9358dc7" => :el_capitan
   end
 
-  if MacOS.version == :high_sierra
+  depends_on "python@2" if MacOS.version <= :snow_leopard
+
+  if MacOS.version == :sierra || MacOS.version == :el_capitan
     patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/21a7e7d/aurora-cli/pants_version_1.4.0_dev20.diff"
-      sha256 "0f4e3dcab78974d43ff5225df68969609587656313f8e495908523f89f6cb0a7"
+      url "https://raw.githubusercontent.com/thinker0/aurora/virtualenv-update/pants.diff"
```

Comment:
Why is the patch needed? Has the bug been reported to upstream?

Reply:
https://issues.apache.org/jira/browse/AURORA-1984

Report Upstream